Dockerfile

```diff
@@ -37,6 +37,10 @@ USER 1001:1001
 
 COPY --from=0 /go/bin/dex /usr/local/bin/dex
 
+# Copy module dependencies for CVE scanning / dependency analysis.
```

Comment:
If preferred, this could be reduced to a single COPY, by preparing the folder structure in the 1st build stage.
However this would make the Dockerfile more complicated.

Let me know if I should change it.